cmd/adsysd/integration_tests/systemdaemons/vfs_mocks.py

```diff
@@ -48,3 +81,20 @@ def Mounted(self, bus_name: str, obj_path: dbus.ObjectPath, display_name: str, s
                 user_visible: bool, mount_location: list[bytes], mount_spec: tuple[list[bytes], dict[str, list[bytes]]],
                 default_location: list[bytes]):
         return
+
+    def get_spec_values(self, protocol: str, mount_spec: tuple[list[bytes], dict[str, list[bytes]]]) -> tuple[str, str]:
+        server, share = '', ''
+
+        if protocol == 'smb-share':
+            server = ''.join([dbus.String(x) for x in mount_spec[1].get("server")])
+            share = ''.join([dbus.String(x) for x in mount_spec[1].get("share")])
+
+        elif protocol == 'nfs':
+            server = ''.join([dbus.String(x) for x in mount_spec[1].get("host")])
+            share = ''.join([dbus.String(x) for x in mount_spec[0]])
```

Comment:
Mind writing some comments for smb-share and nfs so that we know what mount_spec looks like? I’m puzzled that for smb-share we always take index 1, and for the share of nfs, it’s index 0, so writing them as comment would really help :)